ContextDesktop/src/test/java/com/github/migi_1/Context/server/TestServerWrapper.java

```diff
@@ -14,12 +14,13 @@
 
 import com.jme3.network.Network;
 import com.jme3.network.Server;
+// import org.mockito.BDDMockito;
 
 /**
  * Test suite for the ServerWrapper class.
  */
 @RunWith(PowerMockRunner.class)
-@PrepareForTest(Network.class)
+@PrepareForTest({ServerWrapper.class, Network.class})
 public class TestServerWrapper {
```

Comment:
Why are all tests in this class commented out? What was the problem that this had to be done? I'm pretty sure that I made the ServerWrapper and its test in such a way that nothing can really break it.